src/test/java/org/project/bot/service/impl/DataCalculateServiceImplTest.java

```diff
@@ -0,0 +1,62 @@
+package org.project.bot.service.impl;
+
+import jakarta.inject.Named;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.time.LocalDate;
+import java.time.Month;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@ExtendWith(MockitoExtension.class)
+class DataCalculateServiceImplTest {
+    @InjectMocks
+    private DataCalculateServiceImpl dataCalculateService;
+
+    @Test
+    @Named("Calculate percent for the first day in year")
```

Comment:
Please pay attention to where `@Named` was imported. You need another annotation.